roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check for usage of deprecated variables
+  debug: msg="{{ __deprecation_header }} {{ item }} is a deprecated variable and will be no longer be used in the next minor release. Please update your inventory accordingly."
+  when:
+  - lookup('env', item) | bool
```

Comment:
Looks like it doesn't correctly check for an inventory variable like this. I'll need to come up with a different approach to check if a variable is defined